core/survey/_survey/surveyAnalysis.js

```diff
@@ -122,3 +122,16 @@ export const getBaseUnitNodeDef =
 
     return baseUnitNodeDef
   }
+
+export const getSamplingNodeDefChild =
+  ({ nodeDefParent, chainUuid }) =>
+  (survey) =>
+    SurveyNodeDefs.getNodeDefChildren(
+      nodeDefParent,
+      true
+    )(survey).find(
+      (nodeDefChild) =>
+        NodeDef.isSampling(nodeDefChild) &&
+        NodeDef.getChainUuid(nodeDefChild) === chainUuid &&
+        !NodeDef.isDeleted(nodeDefChild)
+    )
```

Comment:
this was in the AreaBasedEstimated component too, better to have it in this "reader"